lib/npm.js

```diff
@@ -51,6 +51,13 @@
 
   npm.commands = {}
 
+  // TUNING
```

Comment:
as in, give us a single place to play around with numbers so we can find a good balance to ship with. We're not gonna make this user-configurable.